port/common/omrport.tdf

```diff
@@ -1631,5 +1631,5 @@ TraceException=Trc_PRT_retrieveLinuxMemoryStats_failedOpeningSwappinessFs Group=
 TraceException=Trc_PRT_retrieveLinuxMemoryStats_failedReadingSwappiness Group=sysinfo Overhead=1 Level=1 NoEnv Template="retrieveLinuxMemoryStats: Failed to read /proc/sys/vm/swappiness. Error code = %d."
 TraceException=Trc_PRT_retrieveLinuxMemoryStats_unexpectedSwappinessFormat Group=sysinfo Overhead=1 Level=1 NoEnv Template="retrieveLinuxMemoryStats: Expected %d items to read, but read %d items."
 
-TraceEntry=Trc_PRT_sysinfo_get_process_start_time_enter Group=sysinfo Overhead=1 Level=1 NoEnv Template="Enter omrsysinfo_get_process_start_time, pid=%llu."
-TraceExit=Trc_PRT_sysinfo_get_process_start_time_exit Group=sysinfo Overhead=1 Level=1 NoEnv Template="Exit omrsysinfo_get_process_start_time, pid=%llu, processStartTimeInNanoseconds=%llu, rc=%d."
+TraceEntry=Trc_PRT_sysinfo_get_process_start_time_enter Group=sysinfo Overhead=1 Level=1 NoEnv Template="Enter omrsysinfo_get_process_start_time, pid=%zu."
+TraceExit=Trc_PRT_sysinfo_get_process_start_time_exit Group=sysinfo Overhead=1 Level=1 NoEnv Template="Exit omrsysinfo_get_process_start_time, pid=%zu, processStartTimeInNanoseconds=%llu, rc=%d."
```

Comment:
Normally we don't change the signature of tracepoints. Instead we mark them as obsolete and add new ones (perhaps with the same name). Since these are so new, I'm not sure that it's worth do so here. @babsingh what do you think?

Reply:
If they are not included in a release, we can modify them. @pshipton for confirmation?

Reply:
I'm fine with modifying them. These haven't even promoted to OpenJ9 yet. 
In theory any project could pick up any SHA of OMR and use to for a release, but I'm not aware of anything other than OpenJ9.